com.unity.cinemachine/Runtime/Debug/CinemachineDebug.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using UnityEngine;
+using System.Collections.Generic;
+using System.Text;
+#if UNITY_EDITOR && CINEMACHINE_UIELEMENTS
+using UnityEditor;
+using UnityEngine.UIElements;
+#endif
+
+namespace Unity.Cinemachine
+{
+    /// <summary>Manages onscreen positions for Cinemachine debugging output</summary>
+    static class CinemachineDebug
+    {
+        static List<StringBuilder> s_AvailableStringBuilders;
+        
+#if UNITY_EDITOR && CINEMACHINE_UIELEMENTS
+        const string k_DebugUIName = "CinemachineDebugUI";
+        static Dictionary<Camera, VisualElement> s_CameraViewRectContainers = new();
+        static GameObject s_UIDocumentHolder;
```

Comment:
This stuff is getting added to the scene.  When you open a new scene, what happens to it?  You need to rethink this.  The idea is that this should be a volatile object, not saved with the scene, and created as-needed.  It will be lost when the scene is unloaded.